02week/blog/css/style.css

```diff
@@ -0,0 +1,173 @@
+.navbar {
+  margin: 2%;
+  display: block;
+  width: 95%;
+  border: 2px solid;
+  border-radius: 10px;
+  padding: 5px;
+}
+
+.h1 {
+  text-align: center;
+  font-size: 200%;
+  font-family:serif;
+}
+
+.roboto {
+  font-family: 'Roboto', sans-serif;
+}
+
+.acmeroboto {
+  font-family: 'Roboto', sans-serif;
```

Comment:
You wouldn't set two font-family properties in the same rule.  the second one would overwrite the first.